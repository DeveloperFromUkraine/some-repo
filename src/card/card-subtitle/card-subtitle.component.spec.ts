import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardSubtitleComponent } from "index";
import { ComponentTest } from "../../../test/test-bed/component";


describe('CardSubtitleComponent', () => {
    let fixture: ComponentFixture<CardSubtitleComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [CardSubtitleComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardSubtitleComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});