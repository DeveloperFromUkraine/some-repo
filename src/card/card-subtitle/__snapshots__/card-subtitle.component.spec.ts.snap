// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSubtitleComponent should match snapshot 1`] = `
<ign-card-subtitle>
  <h4
    _ngcontent-c0=""
    class="card-subtitle"
  />
</ign-card-subtitle>
`;
